client/src/TenderList.js

```diff
@@ -15,14 +16,18 @@ const TendersList = () => {
 	});
 	const [error, setError] = useState(null);
 	const navigate = useNavigate();
+	const role = localStorage.getItem("userType");
 
-	const fetchTenders = async (page) => {
+	const fetchTenders = useCallback(async (page) => {
 		setLoading(true);
 		try {
-			const data = await get(`/api/tenders?page=${page}`);
-			if (data && data.results && data.pagination) {
-				setTenders(data.results);
-				setPagination(data.pagination);
+			const tenderData = await get(`/api/tenders?page=${page}`);
+			const bidsData = await get("/api/bidder-bid?page=1");
+
+			if (tenderData && bidsData) {
```

Comment:
I think `tenderData` && `bidData` can't be false, so this condition is unnecessary